deeprank/learn/NeuralNet.py

```diff
@@ -975,10 +974,14 @@ def _plot_scatter_reg(self, figname):
         yvalues = np.array([])
 
         for l in labels:
-
+            
             if l in self.data:
-
-                targ = self.data[l]['targets'].flatten()
+                try:
+                    targ = self.data[l]['targets']
```

Comment:
Just to comfirm: the `targets` data does not need to be `flatten()`, right? I see the old code is using `targ = self.data[l]['targets'].flatten()`